internal/olm/client/status.go

```diff
@@ -109,6 +111,9 @@ func (s Status) HasInstalledResources() (bool, error) {
 			}
 		}
 	}
+	if existOLM {
+		return true, nil
+	}
```

Comment:
I think we need to cover it with tests as well.

Reply:
OK, I will add it.